src/fswAlgorithms/imageProcessing/centerOfBrightness/centerOfBrightness.cpp

```diff
@@ -65,11 +65,6 @@ void CenterOfBrightness::UpdateState(uint64_t CurrentSimNanos)
     cobBuffer = this->opnavCOBOutMsg.zeroMsgPayload;
```

Comment:
:small_red_triangle_down: I'd squash this commit in with 74b9777 also.

Reply:
Done